pim-tool-back/src/main/java/vn/elca/training/model/entity/Group.java

```diff
@@ -0,0 +1,66 @@
+package vn.elca.training.model.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "group_table")
```

Comment:
[N] "Group" is a keyword in SQL. However, there is a way to allow naming the table "Group".

https://thorben-janssen.com/hibernate-tips-escape-table-column-names/